Maps/Moki Town/HeroHome.gd

```diff
@@ -37,27 +37,12 @@ func interaction(node):
 	if node == $Floor2/Self:
 		Floor2SelfDialoge()
 func Floor2TVDialoge():
-	newdialog([
-	"There's an ad for a new video game on TV.",
-	"It's a Pokémon battle simulation game",
-	"called \"Red and Blue Version\"",
-	"...Okay, time to go!"
-	], false)
+	newdialog(tr("INTERACT_TV"), false)
 func consoleDialoge():
-	var text = [
-	"It's a Nintendo Wii U.",
-	"But the new Nintendo Switch is better."
-	]
-	newdialog(text,false)
+	newdialog(tr("INTERACT_CONSOLE"),false)
 	pass
 func Floor2SelfDialoge():
-	newdialog([
-	"It's crammed full of books about Pokémon",
-	"\"Me and my Owten\",",
-	"\"Jerbolta's Big Adventure\",",
-	"\"Quest for the Legends\".",
-	"I've read all of these many times."
-	], false)
+	newdialog(tr("INTERACT_BOOKSHELF"), false)
 func newdialog(var text, var forceArrow):
 	dialog = dialogBox.instance()
 	dialog.loadText(text, forceArrow)
```

Comment:
Should follow Godot's coding standards. Use `new_dialog` and `force_arrow`

Reply:
Oh yeah, I forgot to apply the Godot codestly to the existing code, my bad :stuck_out_tongue:

Reply:
Fixed

Reply:
Fixed